### ISSUE TITLE
Mask undocumented ketos functions

### ISSUE THREAD

**Issue by burke** [Shopify/shadowenv#62]
There are a bunch of built-in functions provided by ketos that we don't document in Shadowlisp in an effort to prevent people from using (and with the intention of removing them someday).

We should actually find a way to mask them and prevent them from being called.

**Comment by Profpatsch** [comment]
Instead of masking stuff, have you considered just copying the [lexer](https://docs.rs/crate/ketos/0.11.1/source/src/ketos/lexer.rs)/[parser](https://docs.rs/crate/ketos/0.11.1/source/src/ketos/parser.rs) modules and reducing them down to the parts necessary for shadowenv?

`ketos` really wants to be a full lisp, but shadowenv needs a tiny subset (only the s-exp parser), and even the lexer can be dumbed down to only do strings basically (or even just binary data).

**Comment by lavoiesl** [comment]
Is it possible to _use_ the modules instead of copying them?

**Comment by Profpatsch** [comment]
Not really, if you copy them and remove everything you don’t need, only the API and a bit of logic should remain.

**Comment by burke** [comment]
Copying them is probably the best idea. I poked around to try to figure out a way to mask them and nothing worked.

I don't have time to do this anytime soon but would happily :+1: a PR that copied in these modules and exposed a more thoughtful API.
